docs/server/source/production-deployment-template/add-node-on-kubernetes.rst

```diff
@@ -127,13 +190,68 @@ Get the file ``bigchaindb-dep.yaml`` from GitHub using:
-the ``BIGCHAINDB_KEYRING`` with a ``:`` delimited list of all the public keys
-in the BigchainDB cluster.
+* Set ``metadata.name`` and ``spec.template.metadata.labels.app`` to the
+  value set in ``bdb-instance-name`` in the ConfigMap, followed by
+  ``-dep``.
+  For example, if the value set in the
+  ``bdb-instance-name`` is ``bdb-instance-0``, set the fields to the
+  value ``bdb-insance-0-dep``.
+
+* Set the value of ``BIGCHAINDB_KEYPAIR_PRIVATE`` (not base64-encoded).
+  (In the future, we'd like to pull the BigchainDB private key from
+  the Secret named ``bdb-private-key``, but a Secret can only be mounted as a file,
+  so BigchainDB Server would have to be modified to look for it
+  in a file.)
+
+* As we gain more experience running BigchainDB in testing and production,
+  we will tweak the ``resources.limits`` values for CPU and memory, and as
+  richer monitoring and probing becomes available in BigchainDB, we will
+  tweak the ``livenessProbe`` and ``readinessProbe`` parameters.
+
+* Set the ports to be exposed from the pod in the
+  ``spec.containers[0].ports`` section. We currently expose 2 ports -
+  ``bigchaindb-api-port`` and ``bigchaindb-ws-port``. Set them to the
+  values specified in the ConfigMap.
+
+* Uncomment the env var ``BIGCHAINDB_KEYRING``, it will pick up the
+  ``:`` delimited list of all the public keys in the BigchainDB cluster from the ConfigMap.
+
+* Authenticate the new BigchainDB instance using the client x.509 certificate with MongoDB. We need to specify the
+  user name *as seen in the certificate* issued to the BigchainDB instance in order to authenticate correctly.
+  Use the following ``openssl`` command to extract the user name from the certificate:
+
+.. code:: bash
+
+   $ openssl x509 -in <path to the bigchaindb certificate> \
+     -inform PEM -subject -nameopt RFC2253
+
+.. code:: bash
+
+   subject= emailAddress=dev@bigchaindb.com,CN=test-bdb-ssl,OU=BigchainDB-Instance,O=BigchainDB GmbH,L=Berlin,ST=Berlin,C=DE
+
+The ``subject`` line states the complete user name we need to use for creating the user in MongoDB.
+Authenticate the new BigchainDB instance by accessing the ``mongo`` shell and adding the BigchainDB client.
+
+.. code:: bash
+
+   $ kubectl --context ctx-1 exec -it <existing-mongodb-host-name> -c mongodb -- /bin/bash
+   $ mongo --host <existing-mongodb-host-name> --port 27017 --verbose --ssl \
+     --sslCAFile /etc/mongod/ssl/ca.pem \
+     --sslPEMKeyFile /etc/mongod/ssl/mdb-instance.pem
+
+   PRIMARY> use admin
+   PRIMARY> db.auth("adminUser", "superstrongpassword")
+
+   PRIMARY> db.getSiblingDB("$external").runCommand( {
+              createUser: 'emailAddress=dev@bigchaindb.com,CN=test-bdb-ssl,OU=BigchainDB-Instance,O=BigchainDB GmbH,L=Berlin,ST=Berlin,C=DE',
+              writeConcern: { w: 'majority' , wtimeout: 5000 },
+              roles: [
+                { role: 'clusterAdmin', db: 'admin' },
+                { role: 'readWriteAnyDatabase', db: 'admin' }
```

Comment:
I just wanted to verify if all the roles are listed over here.
(I think we added one more role here?)